docs/docs.go

```diff
@@ -125,6 +125,78 @@ const docTemplate = `{
                 }
             }
         },
+        "/info/analysis/status/by/analysis-id/{analysis-id}": {
```

Comment:
How do you like working with `swaggo/swag`?

Reply:
It's not bad. Adding the doc comments that get parsed are a bit verbose, but it's still preferable (to me) over handwriting yet more YAML. I'm definitely burned out on YAML.